Min-Max_Guide.md

```diff
@@ -355,7 +354,7 @@ Another alternative is to exploit RNG to farm clay, but this is illegal in our r
 - Give the rock to Willy and claim the reward for the "How To Win Friends" quest (100g).
 - Place the chest on the bottom-right hand corner of the pier.
 - Fish facing right from the pier. Use maximum casts.
-- At 8:40 AM, destroy the Bamboo Pole and check the rest of the beach for any foragable items + artifact spots.
```

Comment:
why are we not destroying the bamboo pole anymore

Reply:
Destroying is an option. But I thought the focus was on ditching all but the hoe and going to get beach stuff.

Reply:
i want to communicate that you don't need the Bamboo Pole anymore.
after  8:40 you will have training rod, and after that the iridium rod

Reply:
There's been some discussion at the forum on whether the Training Rod is helpful in all cases to all players, and am still open on the question. For me personally in my tests, the Bamboo Pole didn't seem to help me get the Fiberglass Rod and bait before 5:00 p.m. But it also didn't seem worse.

How that works is that some of us don't get significantly more perfect catches with the Training Rod.

Reply:
ok maybe it should go like this then:

bamboo pole until 8:40
--> training rod until level 2
--> bambo pole until enough money for iridium rod
--> iridium rod

Reply:
I like that (assuming you meant Fiberglass Rod). Very creative.

Reply:
yeah i meant Fiberglass